src/components/ArticlesList.jsx

```diff
@@ -3,14 +3,19 @@ import Loading from "./Loading";
 import ErrorMessage from "./ErrorMessage";
 import ArticleCard from "./ArticleCard";
 import styles from "../css/ArticlesList.module.css";
-import { useParams } from "react-router-dom";
+import { useParams, useSearchParams } from "react-router-dom";
 
 function ArticlesList() {
   const { topic } = useParams();
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sortQuery = searchParams.get("sort_by") ?? "created_at";
```

Comment:
we also want to allow the user to toggle between order asc and desc as part of this ticket. Please add this functionality to your app.